src/cljs/main/broadfcui/endpoints.cljs

```diff
@@ -564,6 +564,11 @@
   {:path "/profile/importstatus"
    :method :get})
 
+(defn tos-get-text [on-done]
```

Comment:
I don't have a strong opinion here and I'll ask since I'm wondering about the analog in terra-ui, but is it worthwhile to add an error handler if the call to sam fails?  What does the ui do if sam is unavailable?

Reply:
After testing the scenario in terra ui, you might want to add a network error handler here.  In terra's case, no text is show but it's still possible to click the "agree" button which I'm sure is not something the lawyers would condone.

Reply:
> After testing the scenario in terra ui, you might want to add a network error handler here. In terra's case, no text is shown but it's still possible to click the "agree" button which I'm sure is not something the lawyers would condone.

NVM! I see you've already properly handled things; my bad!